scripts/variantstore/variant_annotations_table/ReadMe.md

```diff
@@ -0,0 +1,107 @@
+
+
+The VAT table is created by that query fresh each time so that there is no risk of duplicates.
+HOWEVER the Genes and VT tables are not. They are cleaned up after 24 hours, but this code needs to be tweaked so that you can’t get into a state where duplicates are created here. The real question here is going to be, is there a use case that we might want to run where adding to a VAT that was created say weeks ago is beneficial, but given that calculations
+
+Note that there are two temporary tables that are created in addition to the main VAT table
+
+
+
+
+
+To check that all of the shards have successfully made it past the first of the sub workflow steps (the most complicated and likely to fail) they will need to be annotated / transformed into json files and put here:
+gsutil ls  [output_path]/annotations/  | wc -l
+
+And then once they have been transformed by the python script and are ready to be loaded into BQ they will be here:
+gsutil ls  [output_path]/genes/  | wc -l
+gsutil ls  [output_path]/vt/  | wc -l
+
+These numbers are cumulative. Also the names of these json files are retained from the original shard names so as to not cause collisions. If you run the same shards through the VAT twice, the second runs should overwrite the first and the total number of jsons should not change.
+Once the shards have make it into the /genes/ and /vt/ directories, the majority of the expense and transformations needed for that shard are complete.
+They are ready to be loaded into BQ. You will notice that past this step, all there is to do is create the BQ tables, load the BQ tables, run a join query and then the remaining steps are all validations or an export into tsv.
+
+
+There are often a fair number of failures from google in this workflow—-so far they have all been 503s. Because of the re-working of the workflow, they should not interrupt unaffected shards, but the shards that are affected will need to be collected and put into a new File of File names and re-run.
+In theory you could actually just re-run the pipeline entirely and since the 503s seem to be completely random and intermittent, the same shards likely wont fail twice and you’d get everything you need, but at low efficiency and high expense.
+To grab any files not in the bucket, but in the file of file names:
+
+gsutil ls [output_path]/genes/ | awk '{print substr($0,90)}' RS='.json.gz\n'  > successful_vcf_shards.txt
+gsutil cat [inputFileofFileNames] | awk '{print substr($0,41)}' RS='.vcf.gz\n' > all_vcf_shards.txt
+comm -3 all_vcf_shards.txt successful_vcf_shards.txt > diff_vcf_shards.txt
+awk '{print g[bucket GVS output VCF shards are in]" $0 ".vcf.gz"}' diff_vcf_shards.txt > fofn_missing_shards.txt
+
+(dont forget to do this for the indices as well)
```

Comment:
Why would a file exist (or not) for a VCF file but not it's corresponding index?